core/executor/src/lib.rs

```diff
@@ -47,6 +47,8 @@ pub use runtime_version::{RuntimeVersion, NativeVersion};
 pub use codec::Codec;
 #[doc(hidden)]
 pub use primitives::Blake2Hasher;
+#[doc(hidden)]
+pub use wasm_interface;
```

Comment:
This is default behavior when you want to use something from a macro, but don't want to show the outside that you have exported it. This will be removed in one of the followings prs anyway.

Reply:
Ah, yeah, would be better with a comment but if you are sure that it would be removed then it is fine!